python/pyspark/pandas/frame.py

```diff
@@ -6318,10 +6318,7 @@ def pivot_table(
                 )
                 psdf = DataFrame(internal)
         else:
-            if isinstance(values, list):
-                index_values = values[-1]
-            else:
-                index_values = values
+            index_values = values
```

Comment:
Hi, @itholic . If we touch the main code, we cannot use `[TEST]` test in the PR title because this is not a test-only change.

Reply:
Thanks! Just changed the title :-)